Source/SelfService/Web/microservice/dataTable.tsx

```diff
@@ -29,8 +29,9 @@ const styles = {
 
 const sortByRuntimeVersion = (params: GridValueGetterParams) => {
     const runtimeVersion = params.row.edit?.extra?.runtimeImage?.replace(/dolittle\/runtime:/gi, '');
+    const capitalize = (string: string) => string.charAt(0).toUpperCase() + string.slice(1);
```

Comment:
This function should be moved outside of the `sortByRuntimeVersion()` body - so that it is not recreated for each call. Probably a very small performance gain - but good practice :)

Also - this fix is possible to do in CSS by setting `text-transform: capitalize` on the component. This _could_ be considered a styling thing - but I'm not entirely sure which one I prefer. I'll let you pick.